sklearn/utils/_testing.py

```diff
@@ -359,6 +362,17 @@ def set_random_state(estimator, random_state=0):
         estimator.set_params(random_state=random_state)
 
 
+def _is_numpydoc():
+    try:
+        import numpydoc
+
+        assert parse_version(numpydoc.__version__) >= parse_version("1.2.0")
```

Comment:
shouldn't we have this in our min_requirements instead of hardcoding the version here?

Reply:
I see it already is in min dep:

https://github.com/scikit-learn/scikit-learn/blob/5600faab452b0adb1f4090219012aa3ae6208c66/sklearn/_min_dependencies.py#L44

So it probably does not need to be included.

I think I was trying to replicate (which has `minversion`):
```python
    pytest.importorskip(
        "numpydoc",
        reason="numpydoc is required to test the docstrings",
        minversion="1.2.0",
    )
```

but since there are not tests, I could also not bother defining this decorator and just use `pytest.importorskip` for the tests? WDYT?

Reply:
yeah that makes sense.